kube/services/revproxy/apply_service

```diff
@@ -4,7 +4,9 @@ scriptDir=$(dirname -- "$(readlink -f -- "$BASH_SOURCE")")
 
 export ARN=$(kubectl get configmap global --output=jsonpath='{.data.revproxy_arn}')
 if [[ ! -z $ARN ]]; then
-  envsubst <$scriptDir/revproxy-service.yaml | kubectl apply -f -
+  envsubst <$scriptDir/revproxy-service-elb.yaml | kubectl apply -f -
 else
   echo "Global configmap not configured"
 fi
+
+kubectl apply -f $scriptDir/revproxy-service.yaml
```

Comment:
let's move apply the old yaml out so it doesn't remove elb before dns is switched